### ISSUE TITLE
"defaults" query in .browserslistrc is not accepted by babel-preset-env

### ISSUE THREAD

**Issue by htulipe** [babel/babel#10965]
## Bug Report

- [x] I would like to work on a fix!

**Current Behavior**
I have a `. browserlistrc` file that contains the `defaults` query. I don't have `targets` option defined in babel-preset-env options. With that setup, the preset includes 148 core-js imports.

If I change this setup and define the `targets` option to `defaults` I would expect the output to be the same. It is not. The preset includes 129  core-js imports.

Note that this happens only with the `defaults` query. If I target for instance `Chrome 75`, both setups work the same.

**Input Code**
See:
https://github.com/htulipe/bug-babel-preset-env

In `lib` you'll find a file compiled without the `targets` option and another with the option defined.

**Expected behavior/code**

I would expect that defining `defaults` in .browserslistrc or in the preset options to ouput the same result.

**Babel Configuration (babel.config.js, .babelrc, package.json#babel, cli command, .eslintrc)**

See sample repo

**Environment**

```
  System:
    OS: macOS 10.15.2
  Binaries:
    Node: 12.13.1 - ~/.nvm/versions/node/v12.13.1/bin/node
    Yarn: 1.21.1 - /usr/local/bin/yarn
    npm: 6.12.1 - ~/.nvm/versions/node/v12.13.1/bin/npm
  npmPackages:
    @babel/cli: ^7.7.7 => 7.7.7
    @babel/core: ^7.7.7 => 7.7.7
    @babel/preset-env: ^7.7.7 => 7.7.7
```

**Comment by JLHwung** [comment]
Closing this as duplicate of #9962 , it is fixed in #10897. However we can not ship it until Babel 8 because it is a breaking change.

As a workaround, you can specify `defaults` again in `targets.browsers`. We recognize that it is not ideal but could not ship in Babel 7 because we are changing the behaviour of default options.

```
[["@babel/preset-env", { "targets": "defaults" }]]
```

@htulipe A personal thank-you for this report and the reproduction repo!

**Comment by htulipe** [comment]
Sorry I missed that @JLHwung, thanks for the link and the workaround.

For those stumbling on the same issue, I got around it by "expliciting" defaults. Meaning I put `> 0.5%, last 2 versions, Firefox ESR, not dead` in my `.browserslist` file. Works fine.
